Source/WebCore/css/query/ContainerQueryFeatures.cpp

```diff
@@ -42,6 +42,15 @@ namespace WebCore::CQ {
 
 using namespace MQ;
 
+enum class UsesSVGZoomRulesForLength : bool { No, Yes };
+
+static LayoutUnit unscaledSizeForPrincipleBox(const Style::PreferredSize& computedSize, LayoutUnit usedSize, UsesSVGZoomRulesForLength usesSVGZoomRulesForLength, const Function<float()>& zoomFunction)
```

Comment:
Why are we not just passing the zoom float directly instead of passing a function to be called?

Reply:
It was to avoid passing a zoom that would go unused but now that I think about it this probably isn't giving us anything beneficial in that regard